src/ReleaseHistory.md

```diff
@@ -5,6 +5,7 @@
 * Bump ELFSharp from 2.13.2 to 2.14.0. [#628](https://github.com/microsoft/binskim/pull/628)
 * Bump System.Reflection.Metadata from 5.0.0 to 6.0.1 and System.Collections.Immutable from 5.0.0 to 6.0.0. [#605](https://github.com/microsoft/binskim/pull/605)
 * Bump ELFSharp from 2.14.0 to 2.15.0. [#631](https://github.com/microsoft/binskim/pull/631)
+* BUGFIX: Unblock running binskim on MacOS. [#576](https://github.com/microsoft/binskim/pull/576)
```

Comment:
>BUGFIX [](http://example.com/codeflow?start=2&length=6)

not sure if this is considerable a bugfix.
maybe `feature: enable BinSkim for MacOS`. or something like that.